packages/terra-icon/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Updated to one-cerner-style-icons v1.21.0
```

Comment:
Should we include:
```
### Added
- IconCircleLinesSymbol
- IconCatalogueNumber
- IconCaution
- IconConsultInstructionsForUse
- IconDateOfManufacture 
- IconManufacturer 
```

Reply:
Updated changelog: https://github.com/cerner/terra-core/pull/2311/commits/acbab240cc223ac16853a5f15fed0d4cb84e749b